rpc/api/proto/mod_service/mod_service.twirp.go

```diff
@@ -1,6 +1,13 @@
-// Code generated by protoc-gen-twirp v8.1.1, DO NOT EDIT.
```

Comment:
seems like we downgraded back to 8.0.0 -- can you re-generate your pb_compiler docker container?